aptos-move/framework/tests/move_unit_test.rs

```diff
@@ -28,11 +28,14 @@ fn run_tests_for_pkg(path_to_pkg: impl Into<String>) {
         ..Default::default()
     };
 
+    let mut utc = UnitTestingConfig::default();
+    utc.filter = test_filter;
```

Comment:
@alinush 
`utc.report_statistics`, if set to true, prints the amount of gas used. Can also be very useful and worth a separate envvar.

Reply:
Ah, yes, can you modify and add a `REPORT_GAS_STATISTICS=1` env. var, and update the `README.md`?